example/rbac/alertmanager/alertmanager-cluster-role.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: alertmanager
+rules:
+- apiGroups: [""]
+  resources:
+  - configmaps
```

Comment:
Actually the Alertmanager pods are not accessing the Kubernetes API at all, so there is no need for a ClusterRole at all.Actually the Alertmanager pods are not accessing the Kubernetes API at all, so there is no need for a ClusterRole at all.